docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,61 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, the enrollment workflow will also guide you to configure syncing of
+user groups and direct application assignments as Access Lists. This will ensure
+that permissions in Okta are modeled properly in Teleport, allowing Teleport users
+to see the same Okta applications that they would when viewing the Okta dashboard.
+
+### Configuring Default Owners
+
+The first step of setting up Okta Access List synchronization is defining default Access List owners. You can select
+any number of default owners or manually enter the owners if they do not yet exist
+in the system. These owners can be later changed and will not be overwritten by the
+Okta Access List synchronization process.
+
+### Importing Okta User Groups as Access Lists
+
+After configurating default owners, you can customize which Okta User Groups should
+be imported. By default, all user groups will be imported. If you would like more
+control, you can disable the "Import All User Groups" option at the top right of
+the panel and configure multiple filters which will be matched against the group
+names. For more information on filter syntax, refer to the Import Filters section
+below.
+
+<Admonition type="note">
+Only Okta User Groups with assignments will be imported as an Access List. If an Okta User
+Group has no assignments, it will not be imported until it has assignments.
+</Admonition>
+
+### Importing Okta Application Assignments as Access Lists
+
+In addition to importing Okta User Groups, you can also import direct application
+assignments within Okta as Access Lists as well. This behaves in exactly the same way
+as user groups.
+
+<Admonition type="note">
+Only Okta Applications with assignments will be imported as an Access List. If an Okta
```

Comment:
I think this would prompt a user to ask a question "what happens if an application assignment is removed, will it remove access list?"